### PR TITLE
Add footerStyle to modal and use Button component

### PR DESCRIPTION
**Pull request by jmophoto**
This adds a footerStyle prop to the modal component and switches the buttons to use the Button component.

This allows for more control of the style of the footer and the ability to add icons to the buttons.

@mxenabled/frontend-engineers 

After (passin in different padding that the default):
screen shot 2016-03-24 at 9 29 39 am

Before:
screen shot 2016-03-24 at 9 38 01 am

### CODE REVIEW

**Comment by cerinman** [comment]
:shipit:

File: demo/app.js [hunk 1]
```diff
@@ -655,6 +657,7 @@ const Demo = React.createClass({
               Footer Content
             </div>
           )}
+          footerStyle={{ padding: '40px 20px' }}
```
Comment (scotttball) [inline demo/app.js]: What's convention on handling inline css or putting it at the bottom and calling it?

Comment (jmophoto) [inline demo/app.js]: Normally we put it all at the bottom and calling it. The app.ls is the demo we use for testing changes and is a little unwieldy.

Comment (cerinman) [inline demo/app.js]: In the demo app file we just inline it.  Not sure why we went that way but I think it was more for ease of setting up demos.  Everywhere else though we always do the style object/function.

Comment (scotttball) [inline demo/app.js]: :+1:

**Pull request merged**